src/index.js

```diff
@@ -70,6 +70,7 @@ async function startApp() {
       const accessToken = req.headers["x-access-token"]
       const userID = req.headers["x-user-id"]
       const timezone = req.headers["x-timezone"]
+      const userAgent = req.headers["user-agent"]
```

Comment:
Will this header _always_ be set? Wondering if this needs some fallback so we're not trying to append to a `null` value later.

Reply:
I can add a fallback.